changelog/3.7.0/3.7.0.md

```diff
@@ -0,0 +1,394 @@
+#### Clustering
+
+- Remote procedure call (RPC) framework for Hybrid mode deployments.
+ [#12320](https://github.com/Kong/kong/issues/12320)
+ [KAG-623](https://konghq.atlassian.net/browse/KAG-623) [KAG-3751](https://konghq.atlassian.net/browse/KAG-3751)
+
+- Dynamic log level over Hybrid mode RPC which allows setting DP log level
+to a different level for specified duration before reverting back
+to the `kong.conf` configured value.
+ [#12320](https://github.com/Kong/kong/issues/12320)
+ [KAG-623](https://konghq.atlassian.net/browse/KAG-623) [KAG-3751](https://konghq.atlassian.net/browse/KAG-3751)
+#### Default
+
+- Added support for debugging with EmmyLuaDebugger.  This feature is a
+tech preview and not officially supported by Kong Inc. for now.
+ [#12899](https://github.com/Kong/kong/issues/12899)
+ [KAG-4316](https://konghq.atlassian.net/browse/KAG-4316)
+
+### Fixes
+#### Configuration
+
+- Fixed default value in kong.conf.default documentation from 1000 to 10000
+for upstream_keepalive_max_requests option.
+ [#12643](https://github.com/Kong/kong/issues/12643)
+ [KAG-3360](https://konghq.atlassian.net/browse/KAG-3360)
+
+- Fix an issue where an external plugin (Go, Javascript, or Python) would fail to
+apply a change to the plugin config via the Admin API.
+ [#12718](https://github.com/Kong/kong/issues/12718)
+ [KAG-3949](https://konghq.atlassian.net/browse/KAG-3949)
+
+- Disable usage of the Lua DNS resolver from proxy-wasm by default.
+ [#12825](https://github.com/Kong/kong/issues/12825)
+ [KAG-4277](https://konghq.atlassian.net/browse/KAG-4277)
+
+- Set security level of gRPC's TLS to 0 when ssl_cipher_suite is set to old
+ [#12613](https://github.com/Kong/kong/issues/12613)
+ [KAG-3259](https://konghq.atlassian.net/browse/KAG-3259)
+#### Core
+
+- Fixed an issue wherein `POST /config?flatten_errors=1` could not return a proper response if the input included duplicate upstream targets
+ [#12797](https://github.com/Kong/kong/issues/12797)
+ [KAG-4144](https://konghq.atlassian.net/browse/KAG-4144)
+
+- **DNS Client**: Ignore a non-positive values on resolv.conf for options timeout, and use a default value of 2 seconds instead.
+ [#12640](https://github.com/Kong/kong/issues/12640)
+ [FTI-5791](https://konghq.atlassian.net/browse/FTI-5791)
+
+- update file permission of kong.logrotate to 644
+ [#12629](https://github.com/Kong/kong/issues/12629)
+ [FTI-5756](https://konghq.atlassian.net/browse/FTI-5756)
+
+- Fixed a problem that in hybrid DP mode a certificate entity configured with vault reference may not get refreshed on time
+ [#12868](https://github.com/Kong/kong/issues/12868)
+ [FTI-5881](https://konghq.atlassian.net/browse/FTI-5881)
+
+- Fix the missing router section for the output of the request-debugging
+ [#12234](https://github.com/Kong/kong/issues/12234)
+ [KAG-3438](https://konghq.atlassian.net/browse/KAG-3438)
+
+- Fixed an issue that leaking locks in the internal caching logic
```

Comment:
I think this is missing a word? "That leaking locks" what?

Reply:
I changed the phrase to "Fixed an issue in the internal caching logic where mutexes could get never unlocked.", wdyt?